front-end-tools/src/Main.tsx

```diff
@@ -1081,19 +1445,974 @@ export default function Main(props: ConnectionProps) {
+                                                    Add your input parameter (
+                                                    {attributeForm.watch('inputParameterType')}):
+                                                </Form.Label>
+
+                                                {attributeForm.watch('inputParameterType') === 'array' && (
+                                                    <textarea
+                                                        {...attributeForm.register('inputParameter')}
+                                                        onChange={(event) => {
+                                                            setParsingErrorRead(undefined);
+                                                            const target = event.target as HTMLTextAreaElement;
+
+                                                            try {
+                                                                JSON.parse(target.value);
+                                                            } catch (e) {
+                                                                setParsingErrorRead((e as Error).message);
+                                                                return;
+                                                            }
+                                                            attributeForm.setValue('inputParameter', target.value);
+                                                        }}
+                                                    >
+                                                        {getArrayExample(entryPointTemplateReadFunction)}
+                                                    </textarea>
+                                                )}
+                                                {attributeForm.watch('inputParameterType') === 'object' && (
+                                                    <textarea
+                                                        {...attributeForm.register('inputParameter')}
+                                                        onChange={(event) => {
+                                                            setParsingErrorRead(undefined);
+                                                            const target = event.target as HTMLTextAreaElement;
+
+                                                            try {
+                                                                JSON.parse(target.value);
+                                                            } catch (e) {
+                                                                setParsingErrorRead((e as Error).message);
+                                                                return;
+                                                            }
+                                                            attributeForm.setValue('inputParameter', target.value);
+                                                        }}
```

Comment:
Can we not just consolidate with the array version of this? What's the difference?

Reply:
Arrays and objects are both treated as JSON objects. 

The difference between both is that we show different examples to the user e.g.
```
export const EXAMPLE_JSON_OBJECT = {
    myStringField: 'FieldValue',
    myNumberField: 4,
    myArray: [1, 2, 3],
    myObject: {
        myField1: 'FieldValue',
    },
};
```
vs.
```
export const EXAMPLE_ARRAYS = 'Examples: \n\n[1,2,3] or \n\n["abc","def"] or \n\n[{"myFieldKey":"myFieldValue"}]';
```

People can then start to edit either the array example or the object example that they choose.